engine/src/main/resources/assets/shaders/lightGeometryPass_frag.glsl

```diff
@@ -126,7 +126,7 @@ void main() {
 
 #if defined (DYNAMIC_SHADOWS) && defined (FEATURE_LIGHT_DIRECTIONAL)
     // ensure that the shadow does not make the surface completely black
-    shadowTerm = clamp(shadowTerm, 0.25, 1.0);
+    shadowTerm = clamp(shadowTerm, 0.5, 1.0);
```

Comment:
I was having troubles seeing under trees.  Even with eye adaptation turned on.  This makes shadows come more in line with the cloud shadows.

Before:
![terasology-160413204259-1280x720](https://cloud.githubusercontent.com/assets/5922109/14516569/3b9766ba-01b9-11e6-85f4-9320621bfa7f.jpg)

After:
![terasology-160413204235-1280x720](https://cloud.githubusercontent.com/assets/5922109/14516572/41116ef6-01b9-11e6-8738-f7f0ce80f094.jpg)